products/dashboard/.env.example

```
@@ -15,3 +15,9 @@ SENTRY_RELEASE=local-dev
 SENTRY_AUTH_TOKEN=sntrys_<token>
 SENTRY_ORG=<org-slug>
 SENTRY_PROJECT=<project-slug>
+
+# PostHog — EU region
+# Project token is public (NEXT_PUBLIC prefix is intentional and correct)
+# Host must always be the EU endpoint — do not change to us.i.posthog.com
+NEXT_PUBLIC_POSTHOG_PROJECT_TOKEN=     # phc_... from PostHog project settings
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)